src/Synchronizer.js

```diff
@@ -88,11 +88,38 @@ module.exports = Class.extend({
          })
          .catch(function(err) {
             self._abortScanning = true;
-            console.log('ERROR: encountered an error while comparing tables', err, err.stack);
+            console.error('Encountered an error while comparing tables', err, err.stack);
          })
          .then(this._outputStats.bind(this));
    },
 
+   trackScanProgress: function(enabled, approxItems, f) {
+      var tick = approxItems > 100 ? 1 : 5,
+          currentPercent = 0,
+          nextProgressPercent = 0,
+          ret;
+
+
+      ret = f(function(currentItems) {
+         currentPercent = Math.round(currentItems / approxItems * 100.0);
+
+         if (currentPercent < 100 && currentPercent >= (nextProgressPercent + tick)) {
+            process.stdout.write('(' + currentPercent + '%)');
+
+            nextProgressPercent =
+               currentPercent >= (nextProgressPercent + tick) ?
+               (nextProgressPercent + tick) :
+               (currentPercent + tick);
+         } else {
+            process.stdout.write('.');
```

Comment:
if you do that, you'll need to think about this part, too

Reply:
I liked the "." so I could see it still working - which is why I wasn't originally printlining it.

Reply:
I agree. I'm just saying that instead of building up a super long line over time, you can use `clearLine()` and `cursorTo(0)` to "overwrite" what was there ... so it would look something like this:

`(1%) ......`

And then when it hits 2%, that would be _replaced with_

`(2%)`

And then the dots would eventually make it

`(2%) ......`